/visualization_node:
  ros__parameters:
    map_frame: map
    base_frame: base_link
    lidar_frame: laser
    planner_frame: planner

    lidar_topic: scan 
    odom_topic: odom 
    datmo_topic: datmo/box_kf 
    filtered_lidar_topic: scan/road_env
    fov_topic: visualization/fov
    obstacles_topic: visualization/obstacles
    
    environment_boundary: [0., 0., 2., 0., 2., -1.5, 0., -1.5]  # CW convex polygon [x1, y1, x2, y2, ...]
    # road_rotation: -1.57079632679
    # road_translation: [1.30, 0.13]
    
    ego_vehicle_size: [0.25, 0.18, 0.12]
    ego_vehicle_offset: [0., 0., 0.06]
    
    # road_xml: /home/christiaan/thesis/robot_ws/install/foresee_the_unseen/share/foresee_the_unseen/resource/road_structure.xml

    planner_frequency: 2. # [Hz]

